config/schedule.rb

```diff
@@ -7,3 +7,8 @@
 every :day, at: '3:00 am', roles: [:job_creator] do
   rake 'clear_expired_sessions'
 end
+
+every Figaro.env.queue_health_check_frequency_seconds.to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.check'
+  runner 'WorkerHealthChecker.enqueue_dummy_jobs'
+end
```

Comment:
Apparently Figaro being present in an initializer may be true, however when running `whenever` directly, the gem uses this file to set the schedule, and does not initialize Rails; therefore the Figaro values are not present. 

1. adding `require File.expand_path(File.dirname(__FILE__) + '/environment')` to schedule.rb loads the Rails environment so that Figaro values are present. 
1. whenever gem expects the interval to be in minutes or higher:
> .rvm/gems/ruby-2.3.1/gems/whenever-0.9.7/lib/whenever/cron.rb:89:in `parse_time': Time must be in minutes or higher (ArgumentError)